admin/recording_uploader/deploy.py

```diff
@@ -10,31 +12,64 @@
 CURRENT_DIR = pathlib.Path(__file__).parent
 
 
-def main(region_name: str = "us-east-1", guided: bool = True) -> None:
+def main(region_name: str = "us-east-1", destroy: bool = False) -> None:
     """Deploy the uploader to AWS Lambda.
 
     Args:
         region_name (str): The AWS region to deploy the Lambda function to.
-        guided (bool): Whether to use the guided SAM deployment.
+        destroy (bool): Whether to delete the Lambda function.
     """
-    s3 = boto3.client(
-        "s3",
-        region_name=region_name,
-        endpoint_url=f"https://s3.{region_name}.amazonaws.com",
-    )
-    bucket = "openadapt"
-
-    s3.create_bucket(
-        ACL="private",
-        Bucket=bucket,
-    )
-
-    # deploy the code to AWS Lambda
-    commands = ["sam", "deploy"]
-    if guided:
-        commands.append("--guided")
-    subprocess.run(commands, cwd=CURRENT_DIR, check=True)
-    logger.info("Lambda function deployed successfully.")
+    # check if aws credentials are set
+    if os.getenv("AWS_ACCESS_KEY_ID") is None:
+        raise ValueError("AWS_ACCESS_KEY_ID is not set")
+    if os.getenv("AWS_SECRET_ACCESS_KEY") is None:
+        raise ValueError("AWS_SECRET_ACCESS_KEY is not set")
+    if destroy:
+        commands = ["sam", "delete", "--no-prompts"]
+    else:
+        s3 = boto3.client(
+            "s3",
+            region_name=region_name,
+            endpoint_url=f"https://s3.{region_name}.amazonaws.com",
+        )
+        bucket = "openadapt"
```

Comment:
What do you think about defining this is config.py?

Reply:
The same reason as above, plus this is hardcoded because this script will be run once in a while (only when the lambda function is changed). And ideally we won't be changing bucket names between runs.